Makefile

```diff
@@ -30,43 +26,20 @@ help: 		   										## Display this help text for Makefile
 .PHONY: upgrade
 upgrade:       										## Upgrade all dependencies to the latest stable versions
 	@echo "=> Updating all dependencies"
-	@if [ "$(USING_PDM)" ]; then $(PDM) update; fi
+	@uv lock --upgrade
 	@echo "=> Python Dependencies Updated"
 	@if [ "$(USING_NPM)" ]; then npm upgrade --latest; fi
 	@echo "=> Node Dependencies Updated"
 	@$(ENV_PREFIX)pre-commit autoupdate
 	@echo "=> Updated Pre-commit"
 
-.PHONY: uninstall
-uninstall:
-	@echo "=> Uninstalling PDM"
-ifeq ($(OS),Windows_NT)
-	@echo "=> Removing PDM from %APPDATA%\Python\Scripts"
-	@if exist "%APPDATA%\Python\Scripts\pdm" (del "%APPDATA%\Python\Scripts\pdm")
-else
-	@echo "=> Removing PDM from ~/.local/bin"
-	@rm -f ~/.local/bin/pdm
-endif
-	@echo "=> PDM removal complete"
-	@echo "=> Uninstallation complete!"
-
 # =============================================================================
 # Developer Utils
 # =============================================================================
-install-pdm: 										## Install latest version of PDM
-	@curl -sSLO https://pdm.fming.dev/install-pdm.py && \
-	curl -sSL https://pdm.fming.dev/install-pdm.py.sha256 | shasum -a 256 -c - && \
-	python3 install-pdm.py
-
```

Comment:
We should add a replacement for this commend.  You can see the updated [`Makefile` here for reference](https://github.com/litestar-org/litestar-vite/blob/main/Makefile)
```Makefile
.PHONY: install-uv
install-uv:                                         ## Install latest version of uv
	@echo "Installing uv..."
	@curl -LsSf https://astral.sh/uv/install.sh | sh >/dev/null 2>&1
	@echo "UV installed successfully"
```

Reply:
added command to makefile